<ui version="4.0" >
 <class>GraphDlg</class>
 <widget class="QDialog" name="GraphDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>GraphDialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>270</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>20</y>
     <width>46</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text" >
    <string>Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>40</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>1.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>70</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>2.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>100</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>3.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>130</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>4.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6" >
   <property name="geometry" >
    <rect>
     <x>150</x>
     <y>20</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Min</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7" >
   <property name="geometry" >
    <rect>
     <x>220</x>
     <y>20</y>
     <width>20</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Max</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8" >
   <property name="geometry" >
    <rect>
     <x>290</x>
     <y>20</y>
     <width>19</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Unit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9" >
   <property name="geometry" >
    <rect>
     <x>380</x>
     <y>20</y>
     <width>25</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Color</string>
   </property>
  </widget>
  <widget class="Line" name="line" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>170</y>
     <width>471</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_10" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>200</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection" >
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text" >
    <string>time range</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="xRangeE" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>200</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>5000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11" >
   <property name="geometry" >
    <rect>
     <x>180</x>
     <y>200</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>ms</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>230</y>
     <width>86</width>
     <height>16</height>
    </rect>
   </property>
   <property name="layoutDirection" >
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text" >
    <string>update frequency</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dtE" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>230</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>50</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13" >
   <property name="geometry" >
    <rect>
     <x>180</x>
     <y>230</y>
     <width>17</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Hz</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14" >
   <property name="geometry" >
    <rect>
     <x>230</x>
     <y>200</y>
     <width>44</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>time ticks</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="xTicksE" >
   <property name="geometry" >
    <rect>
     <x>290</x>
     <y>200</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16" >
   <property name="geometry" >
    <rect>
     <x>230</x>
     <y>230</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>data ticks</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="yTicksE" >
   <property name="geometry" >
    <rect>
     <x>290</x>
     <y>230</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>8</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GraphDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GraphDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
